packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { AlertBar } from '../AlertBar';
+import { ButtonIconReload } from '../ButtonIcon';
+import { useAlertBar } from 'fxa-settings/src/lib/hooks';
+import { useAccount, useLazyAccount } from '../../models';
+import { LinkExternal } from 'fxa-react/components/LinkExternal';
+
+const UTM_PARAMS =
+  '?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices';
+const DEVICES_SUPPORT_URL =
+  'https://support.mozilla.org/kb/fxa-managing-devices' + UTM_PARAMS;
+
+export const Service = () => {
+  return (
+    <section
+      className="my-1"
+      id="service"
+      data-testid="settings-connected-service"
+    >
+      <div className="border-2 border-solid border-grey-100 rounded flex justify-around items-center">
+        <span>icon</span>
+
+        <div className="flex flex-col">
+          <h2>Service Name</h2>
+          <p className="text-sm">location</p>
+        </div>
+
+        <p>Active Now</p>
+
+        <button
+          className="cta-neutral cta-base disabled:cursor-wait whitespace-no-wrap"
+          data-testid="connected-service-sign-out"
+        >
+          Sign out
+        </button>
+      </div>
+    </section>
+  );
+};
+
+export const ConnectedServices = () => {
+  const alertBar = useAlertBar();
+  const [errorText, setErrorText] = useState<string>();
+  const onError = (e: Error) => {
+    setErrorText(e.message);
+    alertBar.show();
+  };
+  const [getAccount, { accountLoading }] = useLazyAccount((error) => {
```

Comment:
This is part of a separate ticket so we don't need to fix it here but I thought our refresh buttons would only ask for the piece being refreshed, but maybe we want to just requery the entire account?

Reply:
Probably just the piece being refreshed, I will address in the refresh ticket

Reply:
Ah, I'm assuming Dave is following the precedent I set in #6423, where I noted

> This doesn't actually perform an update query on the specific section, but simply performs an account retrieval and updates the cache accordingly. I could be persuaded to set up individual queries that narrow the columns we look up, but it's all bundled in to one request anyway so 🤷

I'm still on the fence about it tbh, but open for discussion.

Reply:
Ah gotcha, it feels like the server is doing extra requests instead of asking for and receiving the requested information, I think all we just need another query asking only for `attachedClients`. Outside the scope of this issue though, I'll create a follow up for us in JIRA where we can discuss too.